### PR TITLE
All sorts of improvements, mostly in handling envs

### PR DESCRIPTION
**Pull request by mlech-reef**

### CODE REVIEW

File: {{cookiecutter.repostory_name}}/app/src/requirements.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ django-csp==3.6
 {% endif -%}
 django-environ==0.4.5
 django-extensions==2.1.6
+django-probes==1.5.0
```
Comment (anuj-reef) [inline {{cookiecutter.repostory_name}}/app/src/requirements.txt:7]: what is this for?

File: {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py [hunk 2]
```diff
@@ -184,10 +172,12 @@
...
+## Sentry
+
 if env('SENTRY_DSN', default=''):
     sentry_logging = LoggingIntegration(
         level=logging.INFO,  # Capture info and above as breadcrumbs
-        event_level=logging.ERROR     # Send error events from log messages
+        event_level=logging.ERROR  # Send error events from log messages
     )
 
     sentry_sdk.init(
```
Comment (anuj-reef) [inline {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py:183]: Can we add a default logging configuration?

**Review by anuj-reef — APPROVED** [review]
Thanks for fixing this.

File: {{cookiecutter.repostory_name}}/app/src/requirements.txt (hunk 1 above)
Comment (mlech-reef) [reply {{cookiecutter.repostory_name}}/app/src/requirements.txt:7]: For `docker-compose exec -T app python manage.py wait_for_database`

File: {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py (hunk 2 above)
Comment (mlech-reef) [reply {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py:183]: What do you mean?

**Pull request merged**